code/modules/mob/mob.dm

```diff
@@ -963,6 +963,8 @@
 		return TRUE
 
 	//range check
+	if(interaction_range == null) //null = "you have no set range" aka "unlimited range", used for the AI and AI cards when they are allowed to interact
+		return TRUE
```

Comment:
use infinity if this is the intention, null should not be used as a value here

Reply:
is infinity an actual value? when i made the code last year i was given the impression by a maintainer that DM doesnt have a specific value for an infinate range

Reply:
It is in a technical sense because we use the IEEE float spec, and that has infinity.
But there's also an infinity define that's not that but really really really big, and functionally the same.

It's really not a good pattern to use null like this, ESPECIALLY implicitly

Reply:
i just wasnt aware that there was an actual infinity define, if thats the case then I'll switch all the null cases over to infinity and not need this null check